native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -2,27 +2,28 @@ import UIKit
 import SharedCode
 
 
-class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, ApplicationContractView {
+class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, ApplicationContractView, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Rather than adding all the different protocols that we want ViewController to extend here, it can be nicer to add extensions to the view controller, where we generally add one extension for the different role we want our class to implement. So here we'd probably have one extension for the UIPickerViewDelegate and UIPickerViewDataSource protocols, and another for the UITableViewDelegate and UITableViewDataSource delegates. This helps keep your code more organised, and in the case of really complicated classes with lots of extensions can allow us to split them across multiple files. Does that make sense?